test/test_ops.py

```diff
@@ -38,6 +43,20 @@ def test_unsupported_dtypes(self, device, dtype, op):
             for sample in samples:
                 op(sample.input, *sample.args, **sample.kwargs)
 
+    # Tests that the function and its (array-accepting) reference produce the same
+    #   values on the tensors from sample_inputs func for the corresponding op.
+    @onlyOnCPUAndCUDA
+    @suppress_warnings
+    @ops(ref_test_ops, allowed_dtypes=(torch.float32, torch.long))
+    def test_reference_testing(self, device, dtype, op):
+        # TODO: We should ideally use these tests for UnaryUfuncInfo as well, see: https://github.com/pytorch/pytorch/pull/59369/
+        if isinstance(op, UnaryUfuncInfo) or isinstance(op, SpectralFuncInfo):
+            self.skipTest("Skipped! UnaryUfuncInfo and SpectralOpInfo have implemented reference checks separately.")
```

Comment:
Since reference checks for Spectral Functions and Unary Functions is implemented separately and this PR isn't sufficient (in temrs of design) to address all the ops of both categories, I've added a skip to ensure that testing checks implemented in this PR aren't used for spectral/unary functions.